packages/common/src/lib/common/tools.ts

```diff
@@ -1917,13 +1971,13 @@ async function getHostInfo(objects, callback) {
         data.Platform = 'OSX';
     }
 
-    const systemConfig = await objects.getObjectAsync('system.config');
-    const systemRepos = await objects.getObjectAsync('system.repositories');
+    const systemConfig: ioBroker.Object = await objects.getObjectAsync('system.config');
+    const systemRepos: ioBroker.Object = await objects.getObjectAsync('system.repositories');
```

Comment:
Once the objects DB has proper types, we should remove these annotations so the objects get their actual type that's inferred from the IDs.